otx/algorithms/segmentation/adapters/openvino/task.py

```diff
@@ -261,14 +263,15 @@ def add_prediction(
                     if label_index == 0:
                         continue
                     current_label_soft_prediction = soft_prediction[:, :, label_index]
-                    class_act_map = get_activation_map(current_label_soft_prediction)
+                    if process_soft_prediction:
+                        current_label_soft_prediction = get_activation_map(current_label_soft_prediction)
```

Comment:
Should this be done for pytorch task as well?

Reply:
Currently, torch sem segm task doesn't dump soft predictions

Reply:
https://github.com/openvinotoolkit/training_extensions/blob/3717b7245c8edbdcae45b57a4fb136d75031adef/otx/algorithms/segmentation/task.py#L291

Reply:
@negvet Could you check above mention?